accelerators/README.md

```diff
@@ -70,16 +70,18 @@ pending relocation).
 If the code has not yet been extracted, it should be in a subdirectory.
 Otherwise, it should be linked as a git submodule.
 
+#### Naming Conventions for IaC
+
 Be sure to follow the technology-specific registry guidelines in the event
```

Comment:
Maybe:

"Please follow the technology-specific registry guidelines below."

Reply:
Can you help me understand what is unclear/undesirable about the current phrasing? 

> follow the technology-specific registry guidelines in the event that they conflict with the guidelines noted below

Reply:
Being a non-native English speaker, I found it a bit confusing, specifically the part: "in the event that they conflict with the guidelines noted below". 

The suggestion above is, in my view, more straightforward.

Reply:
I think your suggestion flips the meaning of the sentence, so definitely could be clearer.  How about something like:

"Please follow the naming guidelines below, unless If the technology-specific guidelines conflict.  If there is a conflict, the technology-specific guidelines should be preferred"